shared/github_oauth.go

```diff
@@ -153,7 +155,24 @@ func (gaci githubAccessControlImpl) IsValidWPTMember() (bool, error) {
 	return isMember, err
 }
 
-// NewGitHubAccessControl returns a GitHubAccessControl for checking the permission of a logged-in GitHub user.
+func (gaci githubAccessControlImpl) IsValidAdmin() (bool, error) {
+	valid, err := gaci.isValidAccessToken()
+	if err != nil {
+		return false, err
+	}
+	if !valid {
+		return false, errors.New("Invalid access token")
+	}
+	key := gaci.ds.NewNameKey("Admin", gaci.user.GitHubHandle)
+	var dst struct{}
+	if err := gaci.ds.Get(key, &dst); err != nil {
+		return false, err
```

Comment:
I am getting `Error checking admin` after copied the session cookie, but I am expecting `Admin only` response. What does `gaci.ds.Get(key, &dst);` returns if my name is not in the datastore?